src/test/integ/composite-escape.test.ts

```diff
@@ -1,83 +1,91 @@
-import vscode from "vscode";
+import { strict as assert } from "assert";
+
 import { NeovimClient } from "neovim";
+import vscode from "vscode";
 
 import {
-    attachTestNvimClient,
-    sendVSCodeCommand,
     assertContent,
-    wait,
+    attachTestNvimClient,
     closeAllActiveEditors,
     closeNvimClient,
-    sendEscapeKey,
     openTextDocument,
+    sendEscapeKey,
     sendInsertKey,
+    sendVSCodeKeys,
+    wait,
 } from "./integrationUtils";
 
 describe("Composite escape key", () => {
     let client: NeovimClient;
     before(async () => {
+        await vscode.workspace.getConfiguration("vscode-neovim").update(
```

Comment:
we should use our `Config` abstraction for all interactions with vscode settings. And eventually we should add more safeguards in that abstraction, because vscode settings can be a source of bugs (no type-checking, data can be garbage if users have messed up settings.json).

Reply:
Oh, this is in an integ test so maybe my comment isn't relevant for this code.